### PR TITLE
Enabling Prisma Vulnerabilities check before release

### PR DESCRIPTION
**Pull request by shooshmand-sol**
### What is the purpose of this change?
Checking that no high/critical reported for the latest EMA's docker image before release 

### How was this change implemented?
Enabling steps to call `prisma_vulnurability_checker.py`

Need to merge PR: https://github.com/SolaceDev/sc-cicd-security-tools/pull/27 first.

### CODE REVIEW

File: .github/workflows/release_scripts/prisma_vulnurability_checker.py [hunk 1]
```diff
@@ -23,7 +23,7 @@ def get_excluded_packages():
 
     exclusions = set()
     for exclusion in whitesource_exclusion_entries:
-        exclusions.add(exclusion['package'])
+        exclusions.add(exclusion['packageName'])
```
Comment (shooshmand-sol) [inline .github/workflows/release_scripts/prisma_vulnurability_checker.py:26]: field name has changed to `packageName` in https://us-east-1.console.aws.amazon.com/dynamodbv2/home?region=us-east-1#item-explorer?table=prisma-excluded-packages&maximize=true

**Review by rprasad-solace — APPROVED** [review]
LGTM

**Review by johnvincentcorpuz — APPROVED** [review]
👍

**Review by antonysavio-sol — APPROVED** [review]
👍

**Pull request merged**